drivers/ieee802154/ieee802154_mcr20a.c

```diff
@@ -202,12 +202,13 @@ bool _mcr20a_write_burst(struct mcr20a_spi *spi, bool dreg, u16_t addr,
 {
```

Comment:
The commit message subject mentions coverity ids, these should be placed in body.
So subject could say "drivers/ieee802154: Fix out of bounds access"
and in body you can mention the coverity numbers

Coverity-CID: 178236
Coverity-CID: 178237

Reply:
Title should be: "drivers/ieee802154: Fix out of bounds access in mcr20a driver"

Reply:
Just noticed that there are two bugs about these issues, these should be mentioned in body like this

Fixes #4592 
Fixes #4591

Reply:
Okay, those bugs didn't exist when I did this. I did ask though on the reflector if there was an automated process to convert Coverity bugs into GitHub issues... I'm guessing this answers my question, "no".